{"uuid": "af9230ca-d0ac-4e29-9620-55684c0e1dca", "children": ["7de1eee9-eb4e-4f44-9abf-44cd690eaeba"], "befores": [{"name": "_xunit_setup_method_fixture_TestUserAuth", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log2024-06-23_18-16-12.log'\n", "trace": "  File \"/Users/stepa_air/PycharmProjects/Python_API_course/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/venv/lib/python3.11/site-packages/_pytest/python.py\", line 820, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/venv/lib/python3.11/site-packages/_pytest/python.py\", line 716, in _call_with_optional_argument\n    func()\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/tests/test_user_auth.py\", line 21, in setup_method\n    response1 = MyRequests.post(\"/user/login\", data=data)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/my_requests.py\", line 10, in post\n    return MyRequests._send(url, data, headers, cookies, 'POST')\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/my_requests.py\", line 37, in _send\n    Logger.add_request(url, data, headers, cookies, method)\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/logger.py\", line 29, in add_request\n    cls._write_log_to_file(data_to_add)\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/logger.py\", line 12, in _write_log_to_file\n    with open(cls.file_name, 'a', encoding='utf-8') as logger_file:\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "steps": [{"name": "POST request to URL '/user/login", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log2024-06-23_18-16-12.log'\n", "trace": "  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/my_requests.py\", line 10, in post\n    return MyRequests._send(url, data, headers, cookies, 'POST')\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/my_requests.py\", line 37, in _send\n    Logger.add_request(url, data, headers, cookies, method)\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/logger.py\", line 29, in add_request\n    cls._write_log_to_file(data_to_add)\n  File \"/Users/stepa_air/PycharmProjects/Python_API_course/lib/logger.py\", line 12, in _write_log_to_file\n    with open(cls.file_name, 'a', encoding='utf-8') as logger_file:\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1719155772290, "stop": 1719155772290}], "start": 1719155772290, "stop": 1719155772290}], "start": 1719155772290, "stop": 1719155772293}